### ISSUE TITLE
Add 'therefore' as a premise type ∴

### ISSUE THREAD

**Issue by colinmegill** [arguman/arguman.org#327]
∴

**Comment by fatiherikli** [comment]
Could you give us more context or examples?

Therefore premises should be the conclusion of the argument, right? I have a few questions in that case:

 - Would an argument contain more than one `therefore`s?
 - How do we approve a `therefore` conclusion as a community in an argument?

**Comment by colinmegill** [comment]
Yes, as conclusions. Excellent questions, certainly. I would imagine a
group of 'because' ie., positive statements would be grouped together in
support of a therefore, that's rather how legal language proceeds (ie.,
wherein)

**Comment by fatiherikli** [comment]
So as I see, the `therefore` should be a premise-group in an argument, not an individual premise-type. It's more clear now in that case. Would be great to having such a feature.

We can plan the feature as user stories. Here is my first tryout.

Creating a conclusion scenario:

    Story: User wants to create a new `therefore` or `conclusion` on an argument
    Clicks new conclusion button
    Picks one or more than one premises from current argument
    Types a title and clicks submit button

A scenario for listing conclusions:

    Story: User wants to navigate on a conclusion
    Conclusions are listed on the sidebar
    Users clicks one of the listed conclusions
    Argument map highlights with a different stroke color the picked premises in conclusion

What do you think?

**Comment by colinmegill** [comment]
Wonderful. Yes, that's what was in my mind as well. It's rather nice to
think of being able to visualize how people 'weight' things, ie., different
people will come to different conclusions using different sets of positive
individual premises, the sidebar would allow them to click through and
compare outcomes.

**Comment by colinmegill** [comment]
This make it in?

**Comment by fatiherikli** [comment]
Sorry, the signal must have failed :)

We are working on testing mechanism. We'll implement this feature once we complete all the scenarios on the main website arguman.org

It is something like this:
https://github.com/pyistanbul/dbpatterns/tree/master/tests/scenarios

I am reopening the issue now. Let's keep in touch here!

**Comment by fatiherikli** [comment]
Btw, here is the prototype app for your feature request:
https://github.com/silogizma/
